tools/ops/policylambda.py

```diff
@@ -135,6 +135,7 @@ def dispatch_render(p, sam):
         return None
     policy_lambda = mu.PolicyLambda(p)
     properties = render_func(p, policy_lambda, sam)
+    properties['Role'] = {str('Fn::Sub'): "arn:aws:iam::${AWS::AccountId}:role/%s" % policy_lambda.role}
```

Comment:
we probably need to detect if its an arn already, the policy supports both.

Reply:
Addressed in the latest revision